src/views/pages/students.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import StudentStore from '../../store/student-store';
+import Student from '../components/student.jsx';
+import StudentPageController from '../../controllers/pages/students';
+
+export default class Models extends ListenerComponent {
+
+    componentWillMount(){
+        StudentPageController.getStudents();
+    }
+
+    getStores() {
+        return [
+            StudentStore
+        ];
+    }
+
+    getState() {
+        return {
+            students: StudentStore.getAllStudents()
+        };
+    }
+
+    render() {
+        return (
+            <div id="students">
+                <h1>All Students</h1>
+                {this.state.students.sort(this.studentSort).map((student) => {
+                    return (
+                        <div key={student.id}>
+                            <Student student={student}  />
+                        </div>
+                    );
+                })}
+            </div>
+        );
+    }
+
+    studentSort(a, b){
+        let splitA, splitB, lastA, lastB;
```

Comment:
It'd be nice if you could stop putting NIT's into the reviews, it's starting to annoy me to have to sift through comments to find ones that I actually care about. Why does it matter if I declared them separately?